src/generated/cloud/storagebatchoperations/v1/src/model.rs

```diff
@@ -0,0 +1,1781 @@
+        self.request_id = v.into();
+        self
+    }
+}
+
+impl wkt::message::Message for CreateJobRequest {
+    fn typename() -> &'static str {
+        "type.googleapis.com/google.cloud.storagebatchoperations.v1.CreateJobRequest"
+    }
+}
+
+/// Message for Job to Cancel
+#[serde_with::serde_as]
+#[derive(Clone, Debug, Default, PartialEq, serde::Deserialize, serde::Serialize)]
+#[serde(default, rename_all = "camelCase")]
+#[non_exhaustive]
+pub struct CancelJobRequest {
+    /// Required. The `name` of the job to cancel.
+    /// Format: projects/{project_id}/locations/global/jobs/{job_id}.
+    #[serde(skip_serializing_if = "std::string::String::is_empty")]
+    pub name: std::string::String,
+
+    /// Optional. An optional request ID to identify requests. Specify a unique
+    /// request ID in case you need to retry your request. Requests with same
+    /// `request_id` will be ignored for at least 60 minutes since the first
+    /// request. The request ID must be a valid UUID with the exception that zero
+    /// UUID is not supported (00000000-0000-0000-0000-000000000000).
+    #[serde(skip_serializing_if = "std::string::String::is_empty")]
+    pub request_id: std::string::String,
+}
+
+impl CancelJobRequest {
+    pub fn new() -> Self {
+        std::default::Default::default()
+    }
+
+    /// Sets the value of [name][crate::model::CancelJobRequest::name].
+    pub fn set_name<T: std::convert::Into<std::string::String>>(mut self, v: T) -> Self {
+        self.name = v.into();
+        self
+    }
+
+    /// Sets the value of [request_id][crate::model::CancelJobRequest::request_id].
+    pub fn set_request_id<T: std::convert::Into<std::string::String>>(mut self, v: T) -> Self {
+        self.request_id = v.into();
+        self
+    }
+}
+
+impl wkt::message::Message for CancelJobRequest {
+    fn typename() -> &'static str {
+        "type.googleapis.com/google.cloud.storagebatchoperations.v1.CancelJobRequest"
+    }
+}
+
+/// Message for deleting a Job
+#[serde_with::serde_as]
+#[derive(Clone, Debug, Default, PartialEq, serde::Deserialize, serde::Serialize)]
+#[serde(default, rename_all = "camelCase")]
+#[non_exhaustive]
+pub struct DeleteJobRequest {
+    /// Required. The `name` of the job to delete.
+    /// Format: projects/{project_id}/locations/global/jobs/{job_id} .
+    #[serde(skip_serializing_if = "std::string::String::is_empty")]
+    pub name: std::string::String,
+
+    /// Optional. An optional request ID to identify requests. Specify a unique
+    /// request ID in case you need to retry your request. Requests with same
+    /// `request_id` will be ignored for at least 60 minutes since the first
+    /// request. The request ID must be a valid UUID with the exception that zero
+    /// UUID is not supported (00000000-0000-0000-0000-000000000000).
+    #[serde(skip_serializing_if = "std::string::String::is_empty")]
+    pub request_id: std::string::String,
```

Comment:
I think it might be time to complete the implementation of #439

Reply:
Ack.